docs/pages/reference/cli.mdx

```diff
@@ -311,6 +319,30 @@ $ tsh recordings ls --from-utc=2022-11-01 --to-utc=2022-11-3
 $ tsh recordings ls --last=6h0m0s
 ```
 
+<Admonition
+  type="note"
+  title="Recorded Sessions Availability"
+  scope={["oss", "enterprise"]} scopeOnly
+>
+
+Recorded sessions are linked from the audit events to session recordings files. Self-hosted high availability Teleport
+deployments with directory storage for audit logs and session recordings can result in unavailable playback. That occurs
+since each auth server has its own audit log and recorded session records.
+
+```code
+$ tsh play c8e1b2c5-322a-4095-89e3-391edfd2da9b
+ERROR: Recording for session c8e1b2c5-322a-4095-89e3-391edfd2da9b not found.
+```
+
+Using a SIEM that combines the audit logs will help consolidate the list of available recordings. Downloading the 
```

Comment:
Is it safe to assume that all readers of this page will be familiar with the acronym SIEM?

Reply:
provided